src/custom/components/Popups/FailedNetworkSwitchPopupMod.tsx

```diff
@@ -43,7 +42,7 @@ export default function FailedNetworkSwitchPopup({
       </div>
       <AutoColumn gap="8px">
         <ThemedText.Body fontWeight={500} color={isUnsupportedNetwork ? theme.text2 : theme.text1}>
-          <Trans>{getErrorMessage()}</Trans>
+          <Trans>{getErrorMessage}</Trans>
```

Comment:
Rename `getErrorMessage` to `errorMessage`?

Reply:
Nice catch